test/functional/func/memoize_spec.lua

```diff
@@ -0,0 +1,120 @@
+local t = require('test.testutil')
+local n = require('test.functional.testnvim')()
+local clear = n.clear
+local exec_lua = n.exec_lua
+local eq = t.eq
```

Comment:
Just FYI: it's not really required to assign these in tests anymore, now that we have the `t` and `n` convention.

Reply:
Sorry, I'm not sure what you mean? If I remove those assignments my test fails, since e.g. `exec_lua` is nil

Reply:
I mean you can call `t.eq`, `n.clear` in the tests instead of declaring aliases here. It's up to you though.